src/libxc/lda_xc_1d_ehwlrg.c

```diff
@@ -26,7 +26,7 @@ lda_xc_1d_ehwlrg_init(xc_func_type *p)
  
   assert(p!=NULL && p->params == NULL); 
   p->params = malloc(sizeof(lda_xc_1d_ehwlrg_params)); 
-#ifdef CUDA 
+#if defined CUDA || defined CUDA_MPIV 
```

Comment:
Is this really necessary? Don't we always have CUDA=True if CUDA_MPIV=True? In other words, a CUDA MPI build could just set -DCUDA. Am I missing something? We don't have to change it, I am just wondering.

Reply:
Ignore this comment.